src/emulate.c

```diff
@@ -1336,12 +1354,14 @@ void rv_step(riscv_t *rv, int32_t cycles)
 void ebreak_handler(riscv_t *rv)
 {
     assert(rv);
+    rv->compressed = false;
```

Comment:
In ebreak case, there are two cases — `ebreak` and `c.ebreak`. Both of them execute function `ebreak_handler`, but their instruction length is different. Assign the value to `rv->compressed` in ebreak case respectively.